iam/snippets/src/main/java/EditRole.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.google.cloud.iam.admin.v1.IAMClient;
+import com.google.iam.admin.v1.Role;
+import com.google.iam.admin.v1.Role.RoleLaunchStage;
+import com.google.iam.admin.v1.UpdateRoleRequest;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+
+// [START iam_edit_role]
+public class EditRole {
+
+  public static void main(String[] args) {
+    // TODO(developer): Replace the variables before running the sample.
+    // Role ID must point to an existing role.
+    String projectId = "your-project-id";
+    String roleId = "your-role-id";
+
+    editRole(projectId, roleId);
+  }
+
+  public static void editRole(String projectId, String roleId) {
+
+    String roleName = "projects/" + projectId + "/roles/" + roleId;
+
```

Comment:
Remove additional spaces here

Reply:
Letting formatter dictate. Linter didn't complain.

Reply:
Maybe the linter is not configured to capture it, but in the interest of having a clean code can we remove these unnecessary lines?

Reply:
Removing the lines is more idiomatic